Sources/UINotificationView.swift

```diff
@@ -19,19 +19,51 @@ open class UINotificationView: UIView {
     /// Will be set by the presenter. Is used to animate the notification view with a pan gesture.
     var topConstraint: NSLayoutConstraint?
     
-    /// The left and right margin of the contents.
-    private let leftRightMargin: CGFloat = 20
-    
     /// The limit of translation before we will dismiss the notification.
     internal let translationDismissLimit: CGFloat = -15
     
     // MARK: UI Elements
+    /// Saved to use for resetting the spacing after an image is shown or hidden.
+    private let containerStackViewDefaultSpacing: CGFloat = 14
+    
+    lazy private var containerStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [self.imageView, self.titlesStackView, self.chevronImageView])
+        stackView.axis = .horizontal
+        stackView.spacing = self.containerStackViewDefaultSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .center
+        return stackView
+    }()
+    
+    lazy private(set) open var titlesStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [self.titleLabel, self.subtitleLabel])
+        stackView.axis = .vertical
+        stackView.spacing = 0
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .fill
+        return stackView
+    }()
+    
     lazy public var titleLabel: UILabel = {
-        let label = UILabel(frame: CGRect.zero)
+        let label = UILabel(frame: .zero)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    lazy public var subtitleLabel: UILabel = {
+        let label = UILabel(frame: .zero)
         label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }()
     
+    lazy public var imageView: UIImageView = {
+        let imageView = UIImageView(frame: .zero)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.layer.cornerRadius = 5
```

Comment:
Setting the contentMode might also be a good thing here. The image in the example is all distorted. 

`scaleAspectFill` will work best in most cases, but having a feature to set the ImageView's contentMode might also be great.

Reply:
Good one, didn't think of that! I'll adjust it.

The imageView is public so a custom override of the `UINotificationView` can adjust the contentMode already 🎉